apps/cms/package.json

```diff
@@ -28,7 +28,7 @@
     "clsx": "^1.2.1",
     "graphql": "^16.6.0",
     "graphql-tag": "^2.12.6",
-    "next": "13.2",
+    "next": "~13.3.0",
```

Comment:
Any reason for "~" here?

Reply:
Good one - pnpm added it by default, will remove